test/api/evaluate.test.ts

```diff
@@ -0,0 +1,85 @@
+import {Evaluator} from '@croct/sdk/evaluator';
+import {evaluate, EvaluationOptions} from '../../src/api';
+
+const mockEvaluate: Evaluator['evaluate'] = jest.fn();
+
+jest.mock(
+    '@croct/sdk/evaluator',
+    () => ({
+        __esModule: true,
+        /*
+     * eslint-disable-next-line prefer-arrow-callback --
+     * The mock can't be an arrow function because calling new on
+     * an arrow function is not allowed in JavaScript.
+     */
```

Comment:
```suggestion
         * eslint-disable-next-line prefer-arrow-callback --
         * The mock can't be an arrow function because calling new on
         * an arrow function is not allowed in JavaScript.
         */
```